internal/cmd/combine_prs.go

```diff
@@ -21,75 +21,90 @@ type RESTClientInterface interface {
 }
 
 // CombinePRsWithStats combines PRs and returns stats for summary output
-func CombinePRsWithStats(ctx context.Context, graphQlClient *api.GraphQLClient, restClient RESTClientInterface, repo github.Repo, pulls github.Pulls, command string) (combined []string, mergeConflicts []string, combinedPRLink string, err error) {
+func CombinePRsWithStats(ctx context.Context, graphQlClient *api.GraphQLClient, restClient RESTClientInterface, repo github.Repo, pulls github.Pulls, command string, dryRun bool) (combined []string, mergeConflicts []string, combinedPRLink string, err error) {
```

Comment:
I think soon would be a good time to create a

```go
type CombineOpts struct {
  Noop bool
  Command string
  Repo github.Repo
  Pulls github.Pulls
}
```

Reply:
Done!